src/shared-components/carousel/arrow/index.tsx

```diff
@@ -8,65 +8,51 @@ import { ArrowContainer, BottomRightAlignedArrowContainer } from './style';
 
 type ArrowProps = {
   bottomRightAlignedArrows?: boolean;
-  prev?: boolean;
-  next?: boolean;
   disabled?: boolean;
-  onClick: () => void;
+  next?: boolean;
+  onClick?: () => void;
+  prev?: boolean;
 };
 
-class Arrow extends React.Component<ArrowProps> {
-  static propTypes = {
-    bottomRightAlignedArrows: PropTypes.bool,
-    prev: PropTypes.bool,
-    next: PropTypes.bool,
-    disabled: PropTypes.bool,
-    onClick: PropTypes.func.isRequired,
-  };
-
-  static defaultProps = {
-    bottomRightAlignedArrows: false,
-    prev: false,
-    next: false,
-    disabled: false,
-  };
-
-  arrowClickHandler = () => {
-    const { onClick } = this.props;
-    onClick();
-  };
-
-  render() {
-    const {
-      bottomRightAlignedArrows = false,
-      prev = false,
-      next = false,
-      disabled = false,
-    } = this.props;
-    const ArrowContainerComponent = bottomRightAlignedArrows
-      ? BottomRightAlignedArrowContainer
-      : ArrowContainer;
+const Arrow = ({
+  bottomRightAlignedArrows = false,
+  disabled = false,
+  next = false,
+  onClick = () => undefined,
```

Comment:
I think we should keep this as required, and without default args, no?

Reply:
I initially left it as it was but the way the slider loads the Arrows makes it throw an error passing null to onClick (where we clearly define the prop in the Slider settings) You can check this in PROD in the console https://radiance-ui.curology.com/?path=/story/components-carousel--one-card

since we have no control of Slider but total control of Arrow I decided to change it with defauld value for the console error to disappear